packages/web/components/Dashboard/Post/PostComment.tsx

```diff
@@ -216,6 +217,37 @@ const PostComment: React.FC<PostCommentProps> = ({
           word-wrap: break-word;
         }
 
+        :global(.comment-body h1) {
+          font-size: 1.5em;
+          font-weight: bold;
+          margin: 0.67em 0 0.67em 0;
+        }
+
+        :global(.comment-body h2),
+        :global(.comment-body h3),
+        :global(.comment-body h4) {
+          font-size: 1.2em;
+          font-weight: bold;
+          margin: 0.5em 0 0.5em 0;
+        }
+
+        :global(.comment-body li) {
+          list-style: inside;
+          list-style-type: disc;
+          margin-left: 20px;
+        }
+
+        :global(.comment-body code) {
+          background-color: #eee;
+          font-family: monospace;
+          padding: 2px;
+        }
+
+        :global(.comment-body blockquote) {
+          border-left: 2px solid #4391C9;
```

Comment:
```suggestion
          border-left: 2px solid ${theme.colors.blueLight};
```

And then `import theme from '@/theme'`